docs/overview/index.md

```diff
@@ -9,64 +9,120 @@ myst:
+- **Plone empowers content editors and web application developers.**
+  The Plone Team includes usability experts who have made Plone easy and attractive for content managers to add, update, and maintain content.
 
+- **Plone is international.**
+  The Plone interface has more than 35 translations, and tools exist for managing multilingual content.
 
-(overview-5-label)=
+- **Plone follows standards and is inclusive.**
+  Plone carefully follows standards for usability and accessibility.
+  Plone is compliant with WCAG 2.1 level AA and aims for ATAG 2.0 level AA.
 
-### Traditional server side rendered HTML pages
+- **Plone is open source.**
+  Plone is licensed under the GNU General Public License, the same license used by Linux.
+  This gives you the right to use Plone without a license fee, and to improve upon the product.
 
+- **Plone is supported.**
+  There are over two hundred active developers in the Plone Development Team around the world, and a multitude of companies that specialize in Plone development and support.
 
-(overview-6-label)=
+- **Plone is extensible.**
+  There is a multitude of add-on products for Plone to add new features and content types.
+  In addition, Plone can be scripted using web standard solutions and open source languages.
 
-### Headless CMS with JavaScript frontend through a REST API
+- **Plone is technology neutral.**
+  Plone can interoperate with most relational database systems—both open source and commercial—and runs on a vast array of platforms, including Linux, Windows, macOS, and BSD.
 
 
-(overview-7-label)=
 
-### Explain frontend-backend terms
+(overview-4-label)=
 
+## High Level Overview for Developers
 
-(overview-8-label)=
+Plone is a content management platform with its backend written in Python.
+It is built on top of the open source Zope web application server and development system. 
+Plone is making use of the pluggable Zope Component Architecture (ZCA) to provide a highly modular and extensible system.
+Throughout its history, Plone has used server-side rendering to generate HTML-based content, with advanced resource management features for adding and bundling CSS and JavaScript. 
+Additionally, Plone's use of the ZCA makes it easy to extend and customize, allowing users to create unique, feature rich websites that are tailored to their specific needs.
 
-### Security
+With the release of Plone 6, you now have the option to choose from two different out-of-the-box supported configurations when setting up a new Plone website.
+The Python-based backend server in Plone can still be used alone to render content server-side and deliver HTML to the browser, a setup that is referred to in the Plone documentation as "Classic UI". 
+This configuration has been supported by Plone since its initial release and is still available in the latest version of the platform. 
+For container-based deployment, only the `plone-backend` image is required, or a derivative image with customizations added. 
```

Comment:
```suggestion
For container-based deployment, this option requires only the `plone-backend` image, or a derivative image with customizations added. 
```

Reply:
I think @davisagli's suggestion rephrases the opening clause and essentially repeats it.

I think it would be easier to split this into two sentences, one to establish the requirement of the image as a base, and one to say it may be modified.

```suggestion
For container-based deployment, only the `plone-backend` image is required.
It may be used as a base image, adding customizations, to make a derivative image.
```

Either this or the original works for me.

Reply:
I'll go with the 2 shorter sentences, was struggling with this from the beginnen how to succinctly point out that you can build upon the base image.